MovieChecker/MovieChecker/RottenTomatoes.m

```diff
@@ -0,0 +1,50 @@
+//
+//  RottenTomatoes.m
+//  MovieChecker
+//
+//  Created by Joseph Goldberg on 1/22/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "RottenTomatoes.h"
+
+static NSString *apiKey = @"jw64knjg769gnmd2zs35ttz6";
+static NSString *homeUrlRoot = @"http://api.rottentomatoes.com/api/public/v1.0.json?apikey=";
+static NSString *baseURL = @"http://api.rottentomatoes.com/api/public/v1.0/";
+static NSString *boxOfficeURL = @"lists/movies/box_office.json?limit=50";
+
+@implementation RottenTomatoes
+
++ (NSURL *)appRootURL
+{
+    NSString *url = [[NSString stringWithString:homeUrlRoot] stringByAppendingString:apiKey];
+    return [NSURL URLWithString:url];
+}
+
++ (NSURL *)boxOfficeURL
+{
+    // TODO: Does this need to be cleaned up?
+    NSString *url = [[NSString stringWithString:baseURL] stringByAppendingFormat:@"%@&apikey=%@", boxOfficeURL,apiKey];
```

Comment:
URLWithString:relativeToURL: seems to only work for paths, and does not append parameters correctly, so leaving this as-is (with the space after the comma added, of course).